apps/tng-dictionary/src/components/VocabularyListIndex/VocabularyListIndex.tsx

```diff
@@ -9,11 +9,10 @@ import { TextField, CircularProgress } from '@mui/material';
 import SearchIcon from '@mui/icons-material/Search';
 import MenuBookTwoToneIcon from '@mui/icons-material/MenuBookTwoTone';
 import doValuesMatchFilters from '../../utilities/doValuesMatchFilters';
-import Stack from '@mui/material/Stack';
 import stringIncludes from '../../utilities/matchers/stringIncludes';
 import { HasIdAndName } from '../../types/HasNameAndId';
-
-
+import Test from '../sandbox/Test/Test';
```

Comment:
Is this used? We should setup rules that give us a lint error if we import modules that are not used. Also, prettier is probably not working for you, because it would remove unused imports automatically.